guide-pt.md

```diff
@@ -0,0 +1,812 @@
+# O Guia
+
+Este é o caminho que recomendo para se aprender Haskell.
+
+#### Uma coisa para se lembrar sempre: *não se preocupe com aquilo que não entender imediatamente*. Apenas continue.
+
+## Comunidade
+
+Nosso canal no IRC é `#haskell-beginners` no Freenode.
+
+Cliente web para IRC [aqui](http://webchat.freenode.net/).
+
+[Listas de e-mails](https://wiki.haskell.org/Mailing_lists) relacionadas a Haskell.
+
+## Diretrizes para a Comunidade
+
+[Leia a postagem de Chris Done sobre ensino](http://chrisdone.com/posts/teaching) (Em inglês)
+
+Seja gentil e cortês. Ser cruel ou rude assusta as pessoas e as desencoraja a participar.
+
+Criticismo barato beneficia apenas a pessoa que o emite, não a pessoa que o recebe.
+
+Não descreva coisas como "fáceis" ou "triviais". Você fará pessoas se sentirem mal por ter que trabalhar duro por progredir naquilo. Pessoas que aprendem lentamente frequentemente são aquelas que aprendem melhor, isso é algo para ser celebrado!
```

Comment:
I believe this could be better put this way:

Você fará pessoas se sentirem mal por terem que trabalhar duro para progredirem.